test/test_perlcritic_linter.vader

```diff
@@ -0,0 +1,42 @@
+" NOTE: We use the 'b:' forms below both to make testing easier (no need to
+" mess with globals) and to ensure that we're properly using ale#Var()
+
+Before:
+  runtime ale_linters/perl/perlcritic.vim
+  silent noautocmd new testfile.pl
```

Comment:
I recommend using `ale#test#SetFilename` instead. `new` produces weird output when you run the Vader tests inside of Vim while editing the tests.